From 61a515ac93756eeb1cda9a519df8a017af0095b7 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 21 Sep 2018 14:39:22 +0800
Subject: [PATCH] arm64: dts: rockchip: fixes the dmc for
 rk3399-excavator-sapphire

This patches fixes the vop bandwidth control on rk3399 excavator
sapphire board.

Fixes: 739d1ab1c990
("arm64: dts: rockchip: enable the dmc for rk3399 sapphire excavator")

Change-Id: Icad42cfc4e883668dc3987f1a5561a781d0591b8
Reported-by: Finley Xiao <finley.xiao@rock-chips.com>
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
index c406e54b4a87..6cb9bcd6d984 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
@@ -150,9 +150,10 @@
 	>;
 	vop-bw-dmc-freq = <
 	/* min_bw(MB/s) max_bw(MB/s) freq(KHz) */
-		0       577      200000
-		578     1701     300000
-		1702    99999    400000
+		0       762      200000
+		763     1893     400000
+		1894    3012     528000
+		3013    99999    800000
 	>;
 	auto-min-freq = <200000>;
 };
-- 
2.35.3

